_search-plugins/star-tree-index.md

```diff
@@ -249,6 +249,115 @@ POST /logs/_search
 ```
 {% include copy-curl.html %}
 
+#### Keyword and numeric terms aggregations
+
+You can use [terms aggregations]({{site.url}}{{site.baseurl}}/aggregations/bucket/terms/) on both keyword and numeric fields with star-tree index search.
+
+For star-tree search compatibility with terms aggregations, remember the following behaviors:
+
+- The fields used in the terms aggregation should be part of dimensions defined in the star-tree index.
+- Metric sub-aggregations are optional as long as the relevant metrics are part of the star-tree configuration.
+
+This example aggregates logs by the `user_id` field and returns the counts for each unique user:
+
+```json
+POST /logs/_search
+{
+    "size": 0,
+    "aggs": {
+        "users": {
+            "terms": {
+                "field": "user_id"
+            }
+        }
+    }
+}
+```
+{% include copy-curl.html %}
+
+This example aggregates orders by the `order_quantity` and calculates the average `total_price` for each quantity:
+
+```json
+POST /orders/_search
+{
+    "size": 0,
+    "aggs": {
+        "quantities": {
+            "terms": {
+                "field": "order_quantity"
+            },
+            "aggs": {
+                "avg_total_price": {
+                    "avg": {
+                        "field": "total_price"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+{% include copy-curl.html %}
+
+#### Range aggregations
+
+You can use [range aggregations]({{site.url}}{{site.baseurl}}/aggregations/bucket/range/) on numeric fields to benefit from star-tree index search capabilities.
```

Comment:
Please rewrite the ending of this sentence for clarity.

Reply:
LGTM - your changes look good.